source/boundary_temperature/dynamic_core.cc

```diff
@@ -303,21 +303,24 @@ namespace aspect
     {
       data_OES.clear();
       if (name_OES.size()==0) return;
-      std::istringstream in(Utilities::read_and_distribute_file_content(name_OES.c_str(), this->get_mpi_communicator()));
+      std::istringstream in(Utilities::read_and_distribute_file_content(name_OES.c_str(),
+                                                                        this->get_mpi_communicator()));
       if (in.good())
         {
           str_data_OES data_read;
-          const int buff_size = 1024;
-          char *line = new char [buff_size];
```

Comment:
important to do a dynamic allocation here. ;-)

Reply:
Yes, one could have done without that here :-)